src/plugins/margin.js

```diff
@@ -1,20 +1,30 @@
 import _ from 'lodash'
```

Comment:
I think it's better to tree-shake the used `lodash` functions instead of importing the whole library

```suggestion
import flatMap from 'lodash/flatMap'
import startsWith from 'lodash/startsWith'
```

Reply:
If you want I can make a PR to do this in every file btw 🙂

Reply:
Since this runs in node and not the browser it doesn't matter honestly. No bundle size to worry about.